components/KatFlips/KatFlips.tsx

```diff
@@ -267,12 +265,16 @@ export function KatFlips(props: Props) {
                 {!isLoggedIn ? <hr /> : ''}
             </div>
             <div style={{ display: 'flex', justifyContent: 'space-between' }}>
-                <Form.Control style={{ width: '49%' }} placeholder="Item name..." onChange={onNameFilterChange} />
-                <Form.Control style={{ width: '49%' }} defaultValue={orderBy.value} as="select" onChange={updateOrderBy}>
+                <Form.Control placeholder="Item name..." onChange={onNameFilterChange} />
+                <Form.Control defaultValue={orderBy.value} as="select" onChange={updateOrderBy}>
                     {SORT_OPTIONS.map(option => (
                         <option value={option.value}>{option.label}</option>
                     ))}
                 </Form.Control>
+                <Form.Control placeholder='Minimum Time' onChange={onMinimumTimeChange} />
+                <Form.Control placeholder='Minimum Profit' onChange={onMinimumProfitChange} />
```

Comment:
There should be some padding on the fields to create spaces between them: 
![grafik](https://user-images.githubusercontent.com/58751503/218299828-513a54c4-62bc-4b5d-9a54-eff448c674f7.png)